assembly/src/ast/parsers/adv_ops.rs

```diff
@@ -89,6 +90,15 @@ pub fn parse_adv_inject(op: &Token) -> Result<Node, ParsingError> {
             2 => AdvInject(InsertHperm),
             _ => return Err(ParsingError::extra_param(op)),
         },
+        "sig_to_stack" => match op.num_parts() {
+            3 => match op.parts()[2] {
+                "rpo_falcon512" => AdvInject(PushSignature {
+                    kind: SignatureKind::RpoFalcon512,
+                }),
+                _ => return Err(ParsingError::invalid_param(op, 1)),
+            },
+            _ => return Err(ParsingError::extra_param(op)),
+        },
```

Comment:
Just noticed that there is a slight inconsistency in the instruction name. I think we should use `push_sig` here instead of `sig_to_stack` as it will be more consistent with how we name other similar instructions (and also will match how we serialize the instruction in `src/ast/nodes/advice.rs`).

Reply:
Good catch! Thanks!